src/world/world_server.h

```diff
@@ -27,6 +27,10 @@
 
 #include "http_server.h"
 
+static constexpr auto kTimeServerTickInterval = 2400ms;
```

Comment:
Do these need to be visible in the header? They were in an anon namespace in world_server.cpp because they were only needed there and wrapped to be inaccessible anywhere else

Reply:
I moved this so I could reference it in the time server but I ended up changing that and didn't need it so I'll move it back.

Reply:
Moved this back but kept the name change so it matches the map constants.